### DISCUSSION TITLE
500K White Balance adjustment

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#947] [General]
Hi,

I really like the Native White Balance option, but indoors it doesn't work with artificial lighting.
The Auto White Balance is good but I prefer my footage 500K "cooler". Is there a way to do that in-camera?

Thank you :)

**Comment by dnewman-gpsw** [comment]
Have you tried using oMWIDE=1 (newer cameras can also do $WIDE=1).  This is the native white balance (wide gamut), but can be combined with auto white balance.

**Reply by lyricsdude** [reply]
Yes I use WIDE=1, but I like it 500K cooler :)

**Reply by dnewman-gpsw** [reply]
There is no command for that.  You might be able to make it cooler using MTRX, but there will be math involved.

**Reply by dnewman-gpsw** [reply]
Adding WARM and COOL extensions to future firmware.

**Reply by lyricsdude** [reply]
thank you so much :)

**Reply by lyricsdude** [reply]
> Adding WARM and COOL extensions to future firmware.

when will the firmware be available? can't wait to test it :)
